data.yml

```diff
@@ -1843,32 +1855,93 @@ components:
+          properties: 
+            lodgementDate:
+              type: string
+              description: The lodgement date of the original deposit. 
+            maturityDate:
+              type: string
+              description: Maturity date for the term deposit. 
+            maturityAmount: 
+              type: string
+              description: Amount to be paid upon maturity. If absent it implies the amount to paid is variable and cannot currently be calculated. 
+            maturityCurrency: 
+              type: string
+              description: If absent assumes AUD. 
+            maturityInstructions: 
+              type: string
+              description: Current instructions on action to be taken at maturity. This includes default actions that may be specified in the terms and conditions for the product e.g. roll-over to the same term and frequency of interest payments. 
+              enum:  
+               - HOLD_ON_MATURITY
+               - PAID_OUT_AT_MATURITY
+               - ROLLED_OVER
+              example: PAID_OUT_AT_MATURITY
         loan:
           type: object
           properties:
             startDate:
               type: string
               description: >-
-                (optional) - Start date for the loan.
+                Start date for the loan.
               example: "2019-10-01"
             endDate:
               type: string
               description: >-
-                (optional) - Date that the loan is due to be repaid in full.
+                Date that the loan is due to be repaid in full.
               example: "2045-10-01"
             repaymentType:
               type: string
               description: >-
-                (optional) - Options in place for repayments. If absent defaults to PRINCIPAL_AND_INTEREST.
+                Options in place for repayments. If absent defaults to PRINCIPAL_AND_INTEREST.
               example: "PRINCIPAL_AND_INTEREST"
               enum:
               - INTEREST_ONLY
               - PRINCIPAL_AND_INTEREST
+            loanAmount:
+              type: string
+              description: Optional loan value. 
+            loanCurrency:
+              type: string
+              description: If absent assumes AUD. 
+            nextInstalmentDate: 
+              type: string
+              description: Next date that an instalment is required. 
             minInstalmentAmount:
               type: string
-              description: >-
-                (optional) - Minimum amount of next instalment.
-              example: "5500"
+              description: Minimum amount of next instalment. 
+            minInstalmentCurrency:
+              type: string
+              description: If absent assumes AUD. 
+            maxRedraw: 
+              type: string
+              description: Maximum amount of funds that can be redrawn. If not present redraw is not available even if the feature exists for the account. 
```

Comment:
Add a comma: "If not present, redraw is not available..."

Reply:
Fixed.